src/components/selection_input/selection_input.ts

```diff
@@ -18,6 +19,12 @@ css/* scss */ `
         right: 7px;
         top: 4px;
       }
+      .o-drag-handle {
```

Comment:
icon does not have the correct color. All the icons should (mostly) have the class `o-button-icon`. Mayeb drag handles couls be even lighter in color ? Not sure.

 Also this one should proabably have a `cursor: move` rather than  a `pointer` IMO